.github/workflows/main.yml

```
@@ -0,0 +1,32 @@
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Install fprettify
+        run: pip install fprettify
+
+      - name: Run fprettify on all Fortran files
+        run: |
+          fprettify -r -l 250 ./obs2ioda-v2/src 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)